src/main/java/it/gov/pagopa/pu/workflow/wf/receiptclassification/iuf/IufReceiptClassificationWFClient.java

```diff
@@ -0,0 +1,91 @@
+package it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf;
+
+import io.temporal.api.common.v1.WorkflowExecution;
+import io.temporal.client.WorkflowStub;
+import it.gov.pagopa.payhub.activities.dto.classifications.Transfer2ClassifyDTO;
+import it.gov.pagopa.pu.workflow.exception.custom.WorkflowInternalErrorException;
+import it.gov.pagopa.pu.workflow.service.WorkflowService;
+import it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.classification.IufReceiptClassificationWFImpl;
+import it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.dto.IufReceiptClassificationForReportingSignalDTO;
+import it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.dto.IufReceiptClassificationForTreasurySignalDTO;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Slf4j
+@Service
+public class IufReceiptClassificationWFClient {
```

Comment:
just call it `IufClassification`
```suggestion
public class IufClassificationWFClient {
```